sdk/rust/oak_tests/src/lib.rs

```diff
@@ -36,7 +36,7 @@ use tonic::{
 // TODO(#544): re-enable unit tests of SDK functionality
 
 /// Uses cargo to compile a Rust manifest to Wasm bytes.
-pub fn compile_rust_wasm(cargo_path: &str, module_wasm_file_name: &str) -> anyhow::Result<Vec<u8>> {
+pub fn compile_rust_wasm(cargo_path: &str, module_wasm_file_name: &str, release: bool) -> anyhow::Result<Vec<u8>> {
```

Comment:
This is not very readable when invoked, could you create something like

```rust
enum Profile {
  Release,
  Debug,
}
```

Reply:
Done